### PR TITLE
feature: added typography

### PR DESCRIPTION
**Pull request by omgaikwad**
added typography:
- Heading Examples
- Paragraph Text
- Text Alignments
- Text Gray
- Text With Primary Color
- Text Strike-Through
- Text Underline

### CODE REVIEW

File: components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.heading1,
+.heading2,
+.heading3,
+.heading4,
+.heading5 {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+}
```
Comment (shuklahimanshu81) [inline components/typography/typography.css:9]: You could have used a common class.

**Review by saadahmadhere — COMMENTED** [review]
Looks good!

**Review by shuklahimanshu81 — COMMENTED** [review]
Everything else good to me.

File: components/typography/typography.css (hunk 1 above)
Comment (omgaikwad) [reply components/typography/typography.css:9]: I did it to avoid adding unnecessary classes

**Pull request merged**